examples/streaming-call-option/artifacts/@openzeppelin/contracts/introspection/ERC165.sol/ERC165.dbg.json

```diff
@@ -1,4 +1,4 @@
 {
```

Comment:
I think it's best to add all `artifacts` or any sort of generated files in `.gitignore`.